testutils/src/main/scala/za/co/absa/enceladus/testutils/e2eSparkRunner/E2ESparkRunner.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.e2eSparkRunner
+
+import org.apache.log4j.{LogManager, Logger}
+import za.co.absa.enceladus.testutils.HelperFunctions
+import za.co.absa.enceladus.testutils.models.ProcessMeasurement
+
+import sys.process._
+import scala.io.Source
+
+object E2ESparkRunner {
+  private val log: Logger = LogManager.getLogger(this.getClass)
+
+  private val stdJarPath: String = "$DCE_JAR_PATH/$DCE_STD_SPT_JAR"
+  private val confJarPath: String = "$DCE_JAR_PATH/$DCE_CNFRM_SPT_JAR"
+  private val testUtilsJarPath: String = "$DCE_JAR_PATH/$TEST_UTILS_JAR"
+
+  private val stdClass = "--class za.co.absa.enceladus.standardization.StandardizationJob"
+  private val confClass = "--class za.co.absa.enceladus.conformance.DynamicConformanceJob"
+  private val compClass = "--class za.co.absa.enceladus.testutils.datasetComparison.ComparisonJob"
+
+  private def runBashCmd(bashCmd: String): String = {
+    (s"echo $bashCmd" #| "bash").!!
+  }
+
+  def main(args: Array[String]): Unit = {
+    val cmd = CmdConfig.getCmdLineArguments(args)
+
+    val standartisation = s"spark-submit ${cmd.sparkConf} $stdClass $stdJarPath ${cmd.standartizationJobConf}"
```

Comment:
Consider putting a newline after each logical block, e.g.
```scala
    log.debug(standartisation)
    log.info("Running Standartization")
    val ProcessMeasurement(stdTime, standartisationRes) = HelperFunctions.calculateTime { runBashCmd(standartisation) }
    log.debug(standartisationRes)
    log.info("Standartization Passed")

    val stdComparison = s"spark-submit ${cmd.sparkConf} $compClass $testUtilsJarPath ${cmd.stdComparisonConf}"
    log.debug(stdComparison)
    log.info("Running Standartization Comparison")
    val stdComparisonRes: String = runBashCmd(stdComparison)
    log.debug(stdComparisonRes)
    log.info("Standartization Comparison Passed")
   ...
```
It will increase readability a little bit.

But if you have an inspiration, you can also put such blocks into separate methods. This will make the code even more readable.

Reply:
I think a method for each would be too much, but I did re-order it. Hope it is better now.